apps-rendering/webpack.config.ts

```diff
@@ -46,6 +46,12 @@ function resolve(
 		alias: {
 			logger: path.resolve(__dirname, `src/logger/${loggerName}`),
 		},
+		// Webpack 5 removed a lot of the nodejs polyfills including Buffer
+		// We rely on Buffer for our bridget thrift client  - need for bridget thrift client
+		fallback: {
```

Comment:
I had to add these fall backs in order to run this locally linking to the updated Bridget package with thrift model updates

Reply:
It seems to just be a local thing. Deployed fine!  So I've removed. If it's an issue on anyone elses machine I can add the polyfill to the README. 

Reference commit here: https://github.com/guardian/dotcom-rendering/pull/8920/commits/9a6ac3b1c006708b0316c871ceb10b4ebd608697